xarray/tests/test_variable.py

```diff
@@ -1842,13 +1842,19 @@ def test_quantile_chunked_dim_error(self):
         with pytest.raises(ValueError, match=r"consists of multiple chunks"):
             v.quantile(0.5, dim="x")
 
+    @pytest.mark.parametrize("compute_backend", ["numbagg", "None"], indirect=True)
     @pytest.mark.parametrize("q", [-0.1, 1.1, [2], [0.25, 2]])
-    def test_quantile_out_of_bounds(self, q):
+    def test_quantile_out_of_bounds(self, q, compute_backend):
+        if compute_backend == "numbagg" and q == -0.1:
+            # This should be changed once https://github.com/numbagg/numbagg/issues/276
+            # is resolved.
+            pytest.skip("numbagg does not raise for negative quantiles >= -1")
```

Comment:
Thanks & sorry. I will fix!

Reply:
Fixed upstream, but can take a day for the caches to clear. So fine to merge this as-is if it's ready before then

Reply:
Thanks! I'll just remove the comment then and the tests get the newest numbagg version, right?

Reply:
Yes!

Reply:
Done https://github.com/pydata/xarray/pull/8684/commits/dd952e403df7401395546688b8d4437b1ee10a6b